source/nodejs/adaptivecards/src/card-elements.ts

```diff
@@ -5002,6 +5013,17 @@ export class ActionSet extends CardElement {
         return this._actionCollection.getResourceInformation();
     }
 
+    /**
+     * @inheritdoc
+     */
+    findObject(node: Node): CardObject | undefined {
+        let target;
+        for(const action of this._actionCollection.items) {
+            target = target ?? action.findObject(node); // recur through Actions
```

Comment:
Won't this override target even when one is found? We should stop the loop immediately if a target was found.

Reply:
The `??` operator will prevent that, but refactored this anyways